dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -1,41 +1,31 @@
-import React, { useState, useEffect, useRef } from 'react'
-import { fbt } from 'fbt-runtime'
-import { useStoreState } from 'pullstate'
-import { ethers, BigNumber } from 'ethers'
-import { get, find } from 'lodash'
-
-import AccountStore from 'stores/AccountStore'
-import TransactionStore from 'stores/TransactionStore'
-import ContractStore from 'stores/ContractStore'
-import ApproveModal from 'components/buySell/ApproveModal'
+import React, { useEffect, useState, useRef } from 'react'
 import AddOUSDModal from 'components/buySell/AddOUSDModal'
-import ErrorModal from 'components/buySell/ErrorModal'
-import DisclaimerTooltip from 'components/buySell/DisclaimerTooltip'
+import ApproveButtonLogic from 'components/buySell/ApproveButtonLogic'
 import ApproveCurrencyInProgressModal from 'components/buySell/ApproveCurrencyInProgressModal'
+import ApproveModal from 'components/buySell/ApproveModal'
+import ErrorModal from 'components/buySell/ErrorModal'
+import SettingsDropdown from 'components/buySell/SettingsDropdown'
+import SwapCurrencyPill from 'components/buySell/SwapCurrencyPill'
+import PillArrow from 'components/buySell/_PillArrow'
 import { currencies } from 'constants/Contract'
-import { providersNotAutoDetectingOUSD, providerName } from 'utils/web3'
+import { ethers } from 'ethers'
+import { fbt } from 'fbt-runtime'
+import withIsMobile from 'hoc/withIsMobile'
 import withRpcProvider from 'hoc/withRpcProvider'
```

Comment:
I think @franckc pointed this out some time ago. Can you undo all the changes that are not necessary for this PR? Solely for the reason that it is hard to look at the `diff` of the PR and understand what changed - if most of the imports are re-arranged.